app/routes/v2/orgs.js

```diff
@@ -54,9 +54,10 @@ const createOrg = async(req, res) => {
     });
 
     if(insertedOrg.result.ok) {
+      delete insertedOrg.ops[0].orgAdminKey;
       return res.status(200).send( insertedOrg.ops[0] );
     } else {
-      req.log.error(insertedOrg);
+      req.log.error(orgName);
```

Comment:
I would log the orgName as well as the result status, ensuring that they keys aren't included. With this change as is, we'll know what org we failed to insert, but not what it failed.